lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/filter/UtcConstraintExtractor.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.plugin.iceberg;
+package io.trino.plugin.base.filter;
```

Comment:
> Move UtcConstraintExtractor to plugin-toolkit

i would probably contain in this commit: moving to a module, adding "Utc" name and javadoc, adding Function<ColumNHandle, Type>